hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/CopyContainerCompression.java

```diff
@@ -51,18 +51,16 @@ public OutputStream wrap(OutputStream output) {
   SNAPPY(CompressorStreamFactory.SNAPPY_FRAMED),
   ZSTD(CompressorStreamFactory.ZSTANDARD);
 
+  private static final Logger LOG = LoggerFactory.getLogger(CopyContainerCompression.class);
+
   private final String compressorFactoryName;
 
+  private static final CopyContainerCompression DEFAULT_COMPRESSION = CopyContainerCompression.NO_COMPRESSION;
```

Comment:
I suggest ordering member variables as:

- `static final`
- `static`
- instance variables

I don't know if there is a PMD rule to enforce that, and even if there is, definitely no need to enable it as part of this change.  Just make these moves to reduce future change where possible, like in this class.